tools/autograd/templates/python_torch_functions.cpp

```diff
@@ -534,4 +499,74 @@ void initTorchFunctions(PyObject* module) {
   }
 }
 
+PyObject* handle_torch_function(PythonArgs &r, PyObject* args, PyObject* kwargs, PyTypeObject &torch_api) {
+  PyObject* torch_api_function =
```

Comment:
You're using raw `PyObject*` pointers here, and that is error prone. In fact, you are leaking references to `torch_api_function` here...

Reply:
This is the first time I've ever done work with the Python C API directly and didn't realize I needed to be extra careful. Can you explain in a little more detail how I should fix that?

Reply:
https://github.com/pytorch/pytorch/pull/27064#discussion_r343177627

Reply:
I believe this is fixed now.